phoenix-assembly/src/build/components/all-common-jars.xml

```diff
@@ -88,12 +93,12 @@
       <directory>${project.basedir}/../phoenix-server/target/</directory>
```

Comment:
Need to change this directory. There is no phoenix-queryserver*.jar included in the lib/ directory in the tarball.